package/yast2-kdump.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Thu Sep  1 12:48:14 CEST 2016 - schubi@suse.de
+
+- AutoYaST: Writing entries of profile <general> and <crash_kernel>
```

Comment:
Maybe I would say: `AutoYaST: Write <general> and <crash_kernel> sections correctly` because that "entries of profile <general>..." looked confusing to me. But, please, feel free to ignore (perhaps just a matter of taste).